integration/kubernetes/confidential/agent_image.bats

```diff
@@ -104,6 +108,7 @@ assert_logs_contain() {
 
 @test "$test_tag Test can launch pod with measured boot enabled" {
 	switch_measured_rootfs_verity_scheme dm-verity
+	remove_kernel_param "agent.container_policy_file"
```

Comment:
Hi @BbolroC,
I think it it would be good for others with some insights this change (as discussed offline).

Reply:
Thanks for the comment. In the setup, the following kernel parameter is added if `SKOPEO` is set:

```
if [ "${SKOPEO:-}" = "yes" ]; then
    add_kernel_params \
	"agent.container_policy_file=/etc/containers/quay_verification/quay_policy.json"
fi
```

which drives an agent to run a skopeo command with `--policy /etc/containers/quay_verification/quay_policy.json`, but in this test case, this does not exist because it is not copied into guest. And in this test, `dm-verity` is involved, so we would get an corruption error if the policy file is copied.

It was the best to me to remove a kernel parameter `agent.container_policy_file` before launching a pod. I look forward to any better solutions for this.

Reply:
Hey Choi, it is probably worth noting at this point that the skopeo tests are pretty flaky, so we've turned them off for other platforms as the path is effectively deprecated and replaced by image-rs, so we should be tidying up this soon-ish anyway and removing that config option.

Reply:
yep, thanks for sharing. :wink:

Reply:
Sorry, should have added that this might explain why we've not seen failures with the other jobs as we are running the skopeo tests. It might be worth considering not bothering with the skopeo variant for s390x and just having crictl and kubernetes (with kubernetes being the priority by a fair way), but sorry if you've already done lots of the skopeo adaptations already!

Reply:
Ah, it is very kind of you. It was good to learn about skopeo and how the test works with a signing mechanism. worth spending. :wink: